.github/workflows/lintdiff.yaml

```
@@ -0,0 +1,68 @@
+name: LintDiff (preview)
```

```suggestion
name: LintDiff (Preview)
```

We have been capitalizing `(Preview)` in existing checks.  I don't feel strongly about capital vs lowercase but I do want to be consistent.